### PR TITLE
kubeadm:fix typo in controlplane.go

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

 /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func runControlPlane() func(c workflow.RunData) error {
 			return errors.New("control-plane phase invoked with an invalid data struct")
 		}
 
-		// if this is not a control-plande node, this phase should not be executed
+		// if this is not a control-planed node, this phase should not be executed
```
Comment (yagonobre) [inline cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go]: Please change to `control-plane`

**Review by yagonobre — COMMENTED** [review]
Thanks, @mysunshine92
/priority backlog

File: cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go (hunk 1 above)
Comment (mysunshine92) [reply cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go]: @yagonobre thanks, i have modified it.
would you like to add ok-to-test and lgtm label?

**Review by yagonobre — COMMENTED** [review]
/ok-to-test
/lgtm

**Comment by neolit123** [comment]
thanks for the PR but this change has to wait after code freeze is lifted.
june 11th.

**Comment by mysunshine92** [comment]
> 
> 
> thanks for the PR but this change has to wait after code freeze is lifted.
> june 11th.

ok, thanks.

**Review by dixudx — APPROVED** [review]

**Review by timothysc — APPROVED** [review]
/approve

**Pull request merged**